webapi/tct-batterystatus-w3c-tests/batterystatus/w3c/battery-plugging-in-charging-manual.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Battery Test: battery not full, charger plugging in</title>
+<meta name="flags" content="interact">
+<script src="../../resources/testharness.js"></script>
+<script src="../../resources/testharnessreport.js"></script>
+<style>
+  #note {
+    background-color: #fef1b5;
+    border: solid 1px #cdab2d;
+    padding: 5px;
+    margin: 15px;
+    display: block;
+  }
+</style>
+
+<h2>Description</h2>
+<p>
+  This test validates that onchargingchange and charging attributes exist and
```

Comment:
Since you've split the tests into 4, relevant description should be split too. And please note this modification in the COPYING file.